corfudb-tools/src/main/java/org/corfudb/compactor/CorfuStoreCompactorMain.java

```diff
@@ -87,17 +96,17 @@ private void doCompactorAction() {
             }
         }
         if (config.isStartCheckpointing()) {
-            checkpoint();
+            DistributedCheckpointer distributedCheckpointer = new ServerTriggeredCheckpointer(CheckpointerBuilder.builder()
+                    .corfuRuntime(corfuRuntime)
+                    .cpRuntime(Optional.of(cpRuntime))
+                    .persistedCacheRoot(config.getPersistedCacheRoot())
+                    .isClient(false)
+                    .build(), corfuStore, distributedCheckpointerHelper.getCompactorMetadataTables());
+            checkpoint(distributedCheckpointer);
```

Comment:
Quick question. Any specific reason for moving distributedCheckpointer as a parameter?

Reply:
It made more sense for testing